src/Toolkit/Toolkit.Maui/Esri.ArcGISRuntime.Toolkit.Maui.csproj

```diff
@@ -17,6 +17,7 @@
         <Nullable>annotations</Nullable>
         <WindowsSdkPackageVersion Condition="$([MSBuild]::VersionGreaterThanOrEquals('$(WinAppSDKVersion)','1.6.240923002'))">10.0.19041.38</WindowsSdkPackageVersion> <!-- Workaround for older .NET SDKs. Not needed with VS17.12+  -->
         <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
+        <NoWarn>$(NoWarn);NU1608</NoWarn>
```

Comment:
Note: There's no way to reconcile all Android package warnings at this point that the AndroidX libraries introduce. Talking to the Android team, merely ignoring this is what they also do and should be ok.